docs/BasicCommunication/SetAppProperties/index.md

```diff
@@ -0,0 +1,105 @@
+## SetAppProperties
+
+Type
+: Function
+
+Sender
+: HMI
+
+Purpose
+: Enable/disable an application and set authentication data
+
+!!! must
+
+1. Send `SetAppProperties`(enabled=true) if webEngine app is installed.
+
+!!!
+
+!!! may
+* Request `UpdateSDL` after setting app properties to enforce SDL to perform PTU.
+* Send `SetAppProperties`(enabled=false) to add not-installed webEngine apps to SDL
+
+!!!
+
+!!! info
+
+* The OEM store uses app data in SDL Server (synchronized from SHAID) for SetAppProperties.
+* If SDL does not know the `appID` it should trigger a PTU.
+
+!!!
+
+### Request
+
+#### Parameters
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|properties|[Common.AppProperties](../../common/structs/#appproperties)|true||
+
+
+### Response
+
+#### Parameters
+
+This RPC has no additional parameter requirements.
+
+### Example Request
+```json
```

Comment:
Please clean up the alignment of this JSON, I left some more specific comments on the `GetAppProperties` JSON.

Reply:
please, see updates in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/187/commits/4ebba070bd78b95c3a00721cc2bcb639a6dfb422